amundsen_application/static/js/components/TableDetail/styles.scss

```diff
@@ -1,53 +1,42 @@
 @import 'variables';
 
 .table-detail {
-  margin: 40px auto 48px;
-  display: flex;
-  flex-direction: row;
-}
-
-/*
-  white-space: nowrap breaks without this width on the row
-*/
-.table-detail .row {
-  width: 100%;
-  margin: 0;
-}
-
+  margin: 64px auto 48px;
```

Comment:
LGTM but why 40px --> 64px?

Reply:
Every other page has 64px of margin-top. I don't think this page needs a shorter margin than everything else.